### PR TITLE
test: add test for importUnmanagedInstance

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Refactored smoke test for unmanaging a  VM and then importing it.
Moved the test class to anew file to prevent Trillian re-running them in case of failures of other tests in `test_vm_life_cycle.py`.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?

Tested with a VMware env having changes from #6384 
```
[root@pr6385-t4175-vmware-67u3-marvin marvin]# nosetests --with-xunit --xunit-file=results.xml --with-marvin --marvin-config=./pr6385-t4175-vmware-67u3-advanced-cfg -s -a tags=advanced --hypervisor=VMware tests/smoke/test_vm_life_cycle_unmanage_import.py 
/usr/local/lib/python3.6/site-packages/paramiko/transport.py:33: CryptographyDeprecationWarning: Python 3.6 is no longer supported by the Python core team. Therefore, support for it is deprecated in cryptography and will be removed in a future release.
  from cryptography.hazmat.backends import default_backend
/usr/local/lib/python3.6/site-packages/paramiko/transport.py:236: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,

==== Marvin Init Started ====

=== Marvin Parse Config Successful ===

=== Marvin Setting TestData Successful===

==== Log Folder Path: /marvin/MarvinLogs/May_13_2022_08_08_47_OI6KDY All logs will be available here ====

=== Marvin Init Logging Successful===

==== Marvin Init Successful ====
=== TestName: test_01_unmanage_vm_cycle | Status : SUCCESS ===

=== Final results are now copied to: /marvin//MarvinLogs/test_vm_life_cycle_unmanage_import_M0XC3L ===

```

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
This is WIP

File: test/integration/smoke/test_vm_life_cycle.py [hunk 1]
```diff
@@ -1671,6 +1671,15 @@ def setUp(self):
...
         ]
 
+    def tearDown(self):
+        try:
+            #Clean up, terminate the created templates
+            cleanup_resources(self.apiclient, self.cleanup)
+
+        except Exception as e:
+            raise Exception("Warning: Exception during cleanup : %s" % e)
+        return
```
Comment (DaanHoogland) [inline test/integration/smoke/test_vm_life_cycle.py:1681]:
```suggestion
    def tearDown(self):
        super(TestUnmanageVM, self).tearDown()
```

File: test/integration/smoke/test_vm_life_cycle.py [hunk 2]
```diff
@@ -1752,6 +1761,31 @@ def test_01_unmanage_vm_cycle(self):
...
+            service=importedVMService,
+            templateid=self.template.id)
+
+        self.assertEqual(
+            self.small_offering.id,
+            imported_vm.serviceofferingid,
+            "Imported VM service offering is different"
+        )
+
+        self.cleanup.append(imported_vm)
```
Comment (DaanHoogland) [inline test/integration/smoke/test_vm_life_cycle.py:1788]:
```suggestion
        self.cleanup.append(imported_vm)

        self.assertEqual(
            self.small_offering.id,
            imported_vm.serviceofferingid,
            "Imported VM service offering is different"
        )
```

**Review by DaanHoogland — COMMENTED** [review]
looks good, safe some cleanup details

File: test/integration/smoke/test_vm_life_cycle.py (hunk 1 above)
Comment (shwstppr) [reply test/integration/smoke/test_vm_life_cycle.py]: Done

File: test/integration/smoke/test_vm_life_cycle.py (hunk 2 above)
Comment (shwstppr) [reply test/integration/smoke/test_vm_life_cycle.py]: Done

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
@blueorangutan test centos7 vmware-67u3

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
@blueorangutan test centos7 vmware-67u3

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Review by rohityadavcloud — APPROVED** [review]

**Review by nvazquez — APPROVED** [review]

**Pull request merged**